File: 백준/레벨Bronze IV/10808. 알파벳 개수/알파벳 개수.cc
#include <bits/stdc++.h>
using namespace std;

int alphabet[26];
int main(void) {
  ios::sync_with_stdio(0);
  cin.tie(0);

  string word;
  cin >> word;

  for (auto ch : word) {
    alphabet[ch - 'a']++;
  }
  
  for (auto count : alphabet) {
    cout << count << " ";
  }
}

File: 백준/레벨Bronze IV/10808. 알파벳 개수/알파벳 개수.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String word = br.readLine().trim();
        int[] alphabet = new int[26];

        for (char ch : word.toCharArray()) {
            alphabet[ch - 'a']++;
        }

        StringBuilder sb = new StringBuilder();
        for (int count : alphabet) {
            sb.append(count).append(" ");
        }

        System.out.println(sb);
    }
}